acloud/resource_cluster.go

```diff
@@ -148,6 +152,7 @@ func resourceClusterUpdate(ctx context.Context, d *schema.ResourceData, m interf
 		return diag.FromErr(err)
 	}
 	if cluster != nil {
+		WaitUntilClusterIsRunning(ctx, m, org, *cluster)
```

Comment:
Do we already support starting/stopping a cluster with update?

If so, we have introduced an issue here, if we stop a cluster.